config/environments/development.rb

```diff
@@ -49,6 +49,8 @@
   # Disable sprockets
   config.assets.enabled = false
 
+  config.logger = ActiveSupport::Logger.new($stdout)
```

Comment:
We weren't actually logging to the console in development.

Reply:
Oh yes I remember I had some logging issues when running it locally... although that was something to do with the forking 🤔

Reply:
I'm a bit confused, as I spun out a new rails app (admittedly Rails 6) to compare default configurations, and that didn't have a logger explicitly configured fro development. I suspect there's another issue at play somewhere else, but it might just be because we're not actually spinning up a server process.